src/platform/forms-system/src/js/components/FormSignature.jsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { setData } from 'platform/forms-system/src/js/actions';
+import set from 'platform/utilities/data/set';
+import TextInput from '@department-of-veterans-affairs/component-library/TextInput';
+import Checkbox from '@department-of-veterans-affairs/component-library/Checkbox';
+
+/**
+ * The text input and checkbox which make up the form signature. It's
+ * recommended to use the Attestation component instead of FormSignature
+ * directly, as that one also includes the gray box and handles the
+ * aria-labelledby appropriately.
+ *
+ * Example usage in formConfig:
+ * preSubmitInfo: {
+ *   CustomComponent: (signatureProps) => (
+ *     <section className="box vads-u-background-color--gray-lightest vads-u-padding-bottom--6 vads-u-padding-x--3 vads-u-padding-top--1px vads-u-margin-bottom--7">
+ *       <h3>Statement of truth</h3>
+ *       <p>I solemnly swear I am up to no good.</p>
+ *       <FormSignature
+ *         {...signatureProps}
+ *         signatureLabel="Secret code name"
+ *       />
+ *     <section/>
+ *   ),
+ * }
+ */
+export const FormSignature = ({
+  signatureLabel,
+  checkboxLabel,
+  formData,
+  setFormData,
+  signaturePath,
+  required,
+  showError,
+  validations,
+  onSectionComplete,
+  ariaLabelledBy,
```

Comment:
Can we remove this prop for now? We'll add back `ariaDescribedBy` when Checkbox is updated